ep/build-velox/src/build_velox.sh

```diff
@@ -65,6 +65,16 @@ for arg in "$@"; do
   esac
 done
 
+function apply_compilation_fixes {
+  current_dir=$1
+  velox_home=$2
+  cp ${current_dir}/compilation.patch ${velox_home}/
+  sudo mkdir -p ${velox_home}/third_party/arrow_patches/
+  sudo cp ${current_dir}/helpers.patch ${velox_home}/third_party/arrow_patches/
+  cd ${velox_home}
+  git apply compilation.patch
```

Comment:
If `git apply` fails, will the build script fail or continue the build even though the patch is not applied? Could you confirm it? Thanks!

Reply:
If it fails, I suppose whether to continue or to stop, the build process will fail anyway. How do you think?

Reply:
not actually, as it may build the original code?

Reply:
I think we can do a check after executing `git apply`. And leave some message to help developers easily figure out why build failed.

Reference: https://askubuntu.com/questions/29370/how-to-check-if-a-command-succeeded

Reply:
Updated. Thanks.